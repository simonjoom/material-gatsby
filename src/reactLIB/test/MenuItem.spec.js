import React from 'react';
import { shallow } from 'enzyme';
import MenuItem from '../src/MenuItem';

let wrapper = shallow(<MenuItem className="more" breadcrumbItem href="/" />);

describe('<MenuItem />', () => {
  test('render', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should render a href if provided', () => {
    expect(wrapper.props().href).toEqual('/');
  });
});
