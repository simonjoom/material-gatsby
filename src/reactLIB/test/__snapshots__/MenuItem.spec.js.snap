// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem /> render 1`] = `
<a
  className="breadcrumb more"
  href="/"
/>
`;
